### DISCUSSION TITLE
建议元信息里面加一个是否允许下载的按钮，

### DISCUSSION THREAD

**Discussion by Dashan-IZ** [AlistGo/alist#5226] [Ideas]
建议元信息里面加一个是否允许下载的按钮，我的alist是部署在公网服务器上的，有些视频，音频，文档，不想让别人保存，所以建议在元信息里加一个是否允许下载的按钮，体验上会好很多，这个功能不难实现，希望能尽快安排上

**Comment by Dashan-IZ** [comment]
这是我的alist地址：https://ucloud.cool/

**Comment by Dashan-IZ** [comment]
希望有用到这个功能的gay友（github）帮我顶上去，让开发者看到，谢谢大家

**Comment by Dashan-IZ** [comment]
> 建议元信息里面加一个是否允许下载的按钮，我的alist是部署在公网服务器上的，有些视频，音频，文档，不想让别人保存，所以建议在元信息里加一个是否允许下载的按钮，体验上会好很多，这个功能不难实现，希望能尽快安排上

我的意思是只让别人在线预览，不要别人下载

**Reply by ermzx** [reply]
真是功功又能能，这种很难实现，而且需求意义不明且没必要，你要是只是想让别人预览或者有一定保密需求有很多服务可选，你可以:
自己搭网页，例如目录浏览
使用第三方服务，腾讯文档一类，不提供下载但能预览
